exporter/elasticsearchexporter/README.md

```diff
@@ -78,7 +78,20 @@ All other defaults are as defined by [confighttp].
 
 ### Queuing
 
-The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. However, the sending queue is currently disabled by default.
+The Elasticsearch exporter supports the common [`sending_queue` settings][exporterhelper]. The sending queue is enabled by default.
+
+Default `num_consumers` is `100`. 
+
+When persistent queue is used, there should be no event loss even on collector crashes.
+
+### Batching
+
+The Elasticsearch exporter supports the common `batcher` settings.
```

Comment:
I wanted to mention that we should link here to the description of the batcher settings in the [exporter helper docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md), but apparently there is no documentation on this yet. :melting_face: 

Linking to the [Config struct](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterbatcher/config.go#L15) is definitely not the best experience for end users.

Reply:
> I wanted to mention that we should link here to the description of the batcher settings in the [exporter helper docs](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md), but apparently there is no documentation on this yet. 🫠

Yes, that's the reason behind the lack of links.

> Linking to the [Config struct](https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterbatcher/config.go#L15) is definitely not the best experience for end users.

I think this is reasonable for now